style.css

```diff
@@ -868,15 +868,22 @@ a:visited .nav-title,
 .comment-metadata a:hover,
 .comment-metadata a.comment-edit-link:focus,
 .comment-metadata a.comment-edit-link:hover,
-.comment-reply-link:focus,
-.comment-reply-link:hover,
 .widget_authors a:focus strong,
 .widget_authors a:hover strong,
 .project-terms a:focus,
 .project-terms a:hover {
 	border-bottom-color: transparent;
 	background-color: #767676;
 	color: #fff;
+	transition: all 0.3s linear;
+}
+
+.comment-reply-link:hover{
+	transition: all 0.2s linear;
+	background-color: #767676;
+	color: #fff;
+	padding: 2px 5px 2px 5px;
```

Comment:
We shouldn't be changing padding on hover as it affects layout and does not offer a high performance transition.

See: [https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/#toc-animating-layout-properties](https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/#toc-animating-layout-properties)